modules/swagger-parser-core/src/main/java/io/swagger/v3/parser/core/models/ParseOptions.java

```diff
@@ -36,6 +37,18 @@ public boolean isResolveFully() {
     public void setResolveFully(boolean resolveFully) {
         this.resolveFully = resolveFully;
     }
+    
+    public boolean isResolveRequestBody() {
+		return resolveRequestBody;
+	}
+    
+    /**
+     * If set to true, will help resolving the requestBody as inline, provided resolve is also set to true.
+     * Default is false because of the existing behaviour.
+     */
+	public void setResolveRequestBody(boolean resolveRequestBody) {
+		this.resolveRequestBody = resolveRequestBody;
+	}
```

Comment:
Would need to use spaces instead of tabs and correct indentation

Reply:
Done!